content/en/platform/corda/5.0/application-networks/creating/mgm/cpi.md

```diff
@@ -174,6 +162,10 @@ Build a CPI using the Corda CLI, passing in your generated MGM CPB and `GroupPol
    {{% /tab %}}
    {{< /tabs >}}
 
+{{< note >}}
+Unlike other CPIs, an MGM CPI does not require a CPB file.
```

Comment:
Technically no CPIs _require_ a CPB file so it's not specific to the MGM's CPI. It makes less sense that a non-MGM virtual node would install a CPI without a CPB, but it is possible.

Reply:
```suggestion
An MGM CPI only requires a `GroupPolicy.json` file. A CPB file is not required. 
```

Reply:
Looks great, thanks! 👍